modules/odc-monitoring-your-project-metrics.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * applications/odc-monitoring-project-and-application-metrics-using-developer-perspective.adoc
+
+[id="odc-monitoring-your-project-metrics_{context}"]
+= Monitoring your project metrics
+
+After you create applications in your project and deploy them, you can use the *Developer* perspective in the web console to see the metrics for your project.
+On the left navigation panel of the *Developer* perspective, click *Monitoring* to see the *Dashboard*, *Metrics*, and *Events* for your project.
+
+.Procedure
+
+* Use the *Dashboard* tab to see graphs depicting the CPU, memory, and bandwidth consumption and network related information, such as the rate of transmitted and received packets and the rate of dropped packets.
```

Comment:
Are these steps purposefully unordered?

Reply:
Yes, technically an user can go either to the Dashboard, Metrics, or Events tab. They do not need to be done in this order.